### ISSUE TITLE
[Bug]: underlining doesn't follow the textAlign

### ISSUE THREAD

**Issue by Yunesss** [grammarly/grammarly-for-developers#258]
### What package(s) are you seeing the problem in?

@grammarly/editor-sdk-react

### SDK version

2.0.6

### Browser(s)

Chrome

### Operating system(s)

macOS

### Steps to reproduce

NextJs 12.3.2

Text align center the textarea

```
import { GrammarlyEditorPlugin } from "@grammarly/editor-sdk-react";

export default function Gram() {
  return (
    <GrammarlyEditorPlugin clientId="client_xxx">
      <textarea rows={10} style={{ textAlign: "center" }}></textarea>
    </GrammarlyEditorPlugin>
  );
}
```

### What happened?

The text gets aligned in the center but the underline is offset to the left.

[screenshot](https://i.imgur.com/Z3iou2q.png)

### What did you expect to happen?

underline matching where the words are.

**Comment by ljhaywar** [comment]
@Yunesss Welcome to the Grammarly for Developers community! And thanks for taking the time to report this issue! 

I did a quick test with vanilla JavaScript, and I was able to reproduce the issue when I centered the text in an `input`: [CodeSandbox](https://codesandbox.io/s/issue-258-text-align-center-5zqjft). I'll report this issue to the team.

**Comment by Yunesss** [comment]
Hi again ! thank you for the quick feedback !
This happens in `input` and `textarea`.
This seems to correct itself when the text wraps in a new line.

**Comment by ljhaywar** [comment]
Thanks for the update! I'll pass that along. The team is working on a fix. I'll report back when it's released. 🙌

**Comment by bgschiller** [comment]
@Yunesss, we just released a new version, [2.2.0](https://developer.grammarly.com/docs/changelog/#_2-2-0-2022-11-10), that includes a fix for this issue. Please give it a try when you get a chance, and let me know if you have any questions :)
